subsys/fs/littlefs_fs.c

```diff
@@ -1056,6 +1070,7 @@ static const struct fs_file_system_t littlefs_fs = {
 	.mkdir = littlefs_mkdir,
 	.stat = littlefs_stat,
 	.statvfs = littlefs_statvfs,
+	.gc = littlefs_gc,
```

Comment:
This should not be default. Most systems do not have this.
I would rather guard this by Kconfig option, selectable by systems that support it, and on included on demand by users that need the call.

Reply:
Guarded new functions by `CONFIG_FILE_SYSTEM_GC`, default `n`.